### PR TITLE
Update to Firefox Beta and include Chrome Dev

### PR DESCRIPTION
**Pull request by smiller171**
This PR updates all pages to refer to Firefox Beta instead of Nightly, and includes Chrome Dev in the readme and the login page.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,14 +15,20 @@ Quickstart
...
 ---------------
-Currently WebAuthn only works in [Firefox's Nightly Build](https://download.mozilla.org/?product=firefox-nightly-latest-ssl&os=osx&lang=en-US)
-After installing:
-1. Open the Firefox advanced preferences at the URL (about:config)[about:config]. These are feature flags for FF Nightly.
+Currently WebAuthn only works in [Firefox's Beta Build](https://download.mozilla.org/?product=firefox-beta-latest-ssl&os=osx&lang=en-US) or [Chrome's Dev 
+Channel](https://www.chromium.org/getting-involved/dev-channel) (U2F tokens only in Chrome)  
+After installing:  
+  
+Firefox:
+1. Open the Firefox advanced preferences at the URL [about:config](about:config). These are feature flags for FF.
```
Comment (jcjones) [inline README.md]: As of 60 Beta, `about:config` edits are no longer needed for Firefox. Both Beta and Nightly "just work." :)

File: templates/login.html [hunk 2]
```diff
@@ -25,15 +25,24 @@
             <div class="login-wrapper">
                 <form class="form-signin">
                     <h2 class="form-signin-heading text-center">WebAuthn.io</h2>                    
-                    <p>Currently, WebAuthn is only available to test on <a href="https://www.mozilla.org/en-US/firefox/channel/desktop/">Firefox's Nightly Build</a></p>
-                    <p>Once you've installed the Nightly Build:</p>
+                    <p>Currently, WebAuthn is only available to test on <a href="https://www.mozilla.org/en-US/firefox/channel/desktop/">Firefox's Beta Build</a> and <a href="https://www.chromium.org/getting-involved/dev-channel">Chrome's Dev Channel</a></p>
+                    <p>Once you've installed a compatible browser:</p>
+                    <p>Firefox Beta:</p>
                     <ol>
                         <li>Open the Firefox advanced preference panel at <a href="about:config">about:config</a></li>
```
Comment (jcjones) [inline templates/login.html]: ditto the above here

File: README.md (hunk 1 above)
Comment (smiller171) [reply README.md]: huh, I could have sworn I needed to change it when I installed FF Beta, but just confirmed in a VM. I'll update the PR

**Comment by smiller171** [comment]
@jcjones I've updated the PR to remove extraneous FF instructions

**Pull request merged**